internal/provider/resources/block.go

```diff
@@ -116,16 +120,111 @@ func (r *BlockResource) Schema(_ context.Context, _ resource.SchemaRequest, resp
 			"workspace_id": schema.StringAttribute{
 				Optional:    true,
 				CustomType:  customtypes.UUIDType{},
-				Description: "Workspace ID (UUID) where the Block is located",
+				Description: "Workspace ID (UUID) where the Block is located. In Prefect Cloud, either the resource or the provider's `workspace_id` must be set in order to manage the Block.",
 			},
 		},
 	}
 }
 
+// copyBlockToModel maps an API response to a model that is saved in Terraform state.
+func copyBlockToModel(block *api.BlockDocument, state *BlockResourceModel) diag.Diagnostics {
```

Comment:
that makes sense, bc the `Create()` method pulls from the `req.Plan`, whereas the `Read()` pulls from `req.State`

i'm down to call it a `model` in the helper (since it's shared - otherwise we'd have to call it something like `planOrStateModel` 😆 ) and then we can use the variable names you mentioned in the logic itself

Reply:
this might be a more clear way to disambiguate in the helper

https://github.com/PrefectHQ/terraform-provider-prefect/pull/196/commits/26e2a761cd306912a49f2d8eafc8e300b4dddefa

Reply:
I like it 👍🏼